Vester/Public/New-VesterConfig.ps1

```diff
@@ -210,31 +210,17 @@ function New-VesterConfig {
                 Default      {$null}
             }
 
-            # TODO: Should probably offload this to a private function
```

Comment:
Glad to see this happened. :)

Reply:
While I'm super happy with how this PR turned out, and how much freaking work I'll be doing with the AST moving forward...I think you found the easter egg I was most excited about. 👌